### DISCUSSION TITLE
Welcome to mrouted Discussions!

### DISCUSSION THREAD

**Discussion by troglobit** [troglobit/mrouted#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bischoff** [comment]
I am Éric Bischoff, QA engineer at SUSE, network teacher in a previous life. I contributed only one small thing 5 years ago, #19 , but I am still very interested in this nice project and I follow it with attention.

**Reply by troglobit** [reply]
Hi Éric!  I remember your contribution :)

A lot has happened in mrouted in the last five years. The 4.x series is a big step for this oldie, the addition of IGMPv3 support (ASM only, obviously), lots of IGMP configuration options added, and Cisco style static routing.  

Take care!
